docs/offboarding.rst

```diff
@@ -0,0 +1,250 @@
+SecureDrop Application Key from the list of available keys.
+
+|select securedrop key|
+
+Double-click the key, and in the **Names and Signature** tab, add a name such
+as "OLD <Your Organization> SecureDrop Submission Key - Do Not Delete - Retired
+<Date>". (This is a local-only change to stop you from mixing up the old and
+new keys).
+
+|edit key name|
+
+Once you have done that, you can delete the original name for the key (but not
+the key itself!), so that the only name you see is "OLD <Your Organization>
+Submission Key - Do Not Delete - Retired <Date>".
+
+|delete key name|
+
+Now :doc:`follow the instructions <generate_submission_key>` to create a PGP
+key on the *Secure Viewing Station*. This will be your new *Submission Key.*
+Copy the fingerprint and new *Submission Public Key* to your *Transfer Device*.
+
+.. |select securedrop key| image:: images/offboard/passwords_keys_sd_key.png
+.. |edit key name| image:: images/offboard/add_key_name.png
+.. |delete key name| image:: images/offboard/delete_key_name.png
+
+On the Admin Workstation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Take the *Transfer Device* with the new *Submission Public Key* and fingerprint
+to your *Admin Workstation*. As you did during the initial install, copy the
+public key, ``SecureDrop.asc``, to the ``install_files/ansible_base/``
+directory, replacing the existing public key file that is there.
+
+Then, from the ``~/Persistent/securedrop`` directory, run
+
+    .. code:: sh
+
+      ./securedrop-admin sdconfig
+
+If the new public key that you placed in ``install_files/ansible_base`` has the
+same name the same as the old public key, ``SecureDrop.asc``, the only part of
+the configuration you will change is the SecureDrop *Submission Key*
+fingerprint, which you will update with the fingerprint of your new key.
+
+Once you have completed the above, run
+
+    .. code:: sh
+
+      ./securedrop-admin install
+
+to push the changes to the server.
+
+You may want to immediately create a test submission, then use a Journalist
+account to log into the *Journalist Workstation* and download it to a *Transfer
+Device*, to verify that your new key is working.
+
+Return to the Secure Viewing Station
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the *Secure Viewing Station,* decrypt the test submission you made to ensure
+that your new key is working properly.
+
+**Do not delete your old submission key!** You'll want to maintain it on the
+*SVS* so that you can still decrypt old submissions that were made before you
+changed keys. If you like, you can **Revoke** the key by selecting the key in
+the **Passwords and Keys** application, opening the **Details** tab,
+highlighting the first key entry and clicking **Revoke**. This also makes
+local-only changes and does not stop you or anyone else from using the key, but
+it is a reminder that your key has changed.
+
+|revoke key|
+
+.. |revoke key| image:: images/offboard/revoke_key.png
+
```

Comment:
One final step might be to make sure that any other extant Admin Workstations have their configs updated, as if they have the old key and run the installer it will overwrite changes.